README.md

```diff
@@ -17,16 +17,73 @@ standard implementation of lightweight Virtual Machines (VMs) that feel and
 perform like containers, but provide the workload isolation and security
 advantages of VMs.
 
+## License
+
+The code is licensed under an Apache 2.0 license.
+See [the license file](LICENSE) for further details.
+
+## Platform support
+
+Kata Containers currently runs on 64-bit systems supporting the following
+technologies:
+
+| Architecture | Virtualization technology |
+|-|-|
+| `x86_64`, `amd64` | [Intel](https://www.intel.com) VT-x, AMD SVM |
+| `aarch64` ("`arm64`")| [ARM](https://www.arm.com) Hyp |
+| `ppc64le` | [IBM](https://www.ibm.com) Power |
+| `s390x` | [IBM](https://www.ibm.com) Z & LinuxONE SIE |
+
+### Hardware requirements
+
+The [Kata Containers runtime](src/runtime) provides a command to
+determine if your host system is capable of running and creating a
```

Comment:
I, personally, think that `kata-runtime check` is not the best way to check whether the host system is capable of running and creating a Kata Container.

As we're 100% Kubernetes focused nowadays, I'd rather see us working on and recommending [NFD](https://github.com/kubernetes-sigs/node-feature-discovery) instead.

But hey, I guess this is the best we have now.

Reply:
I find the tool useful as a starting point, however -- most people will come here to test in their VM environment, and this helps. I do wonder if we need /want this here versus user/developer guide?